lib/class-wp-json-comments-controller.php

```diff
@@ -191,6 +167,115 @@ public function delete_item( $request ) {
 		return array( 'message' => __( 'Deleted comment' ) );
 	}
 
+
+	/**
+	 * Check if a given request has access to read comments
+	 * 
+	 * @param  WP_JSON_Request $request Full details about the request.
+	 * @return bool
+	 */
+	public function get_items_permissions_check( $request ) {
+
+		// If the post id isn't specified, presume we can create
```

Comment:
@joehoyle I think I understand where you are coming from.  `$post_id` is a required parameter for creating a comment, but not for returning a collection of comments.  
The issue is what to do in the following use-case:

Jane wants to sent a POST request to create a comment.
Jane first sends an OPTIONS request to `wp/comments` 
Jane expects to receive a response to her request that includes if she CAN create a comment.

@joehoyle Is that correct?